release/models/vlan/openconfig-vlan.yang

```diff
@@ -106,6 +106,36 @@ module openconfig-vlan {
       description "Admin state of the VLAN";
     }
 
+    // Private VLAN
+    leaf private-vlan-type {
```

Comment:
It might be better to define private-vlan-type in openconfig-vlan-types.yang instead. Then, simply refer to it here using `type oc-vlan-types:private-vlan-type`.

Reply:
> It might be better to define private-vlan-type in openconfig-vlan-types.yang instead. Then, simply refer to it here using `type oc-vlan-types:private-vlan-type`.

Sure. I will make this change.